README.md

```diff
@@ -289,7 +289,7 @@ Support this project by becoming a sponsor. Your logo will show up here with a l
 <a href="https://opencollective.com/bpkg/sponsor/7/website" target="_blank"><img src="https://opencollective.com/bpkg/sponsor/7/avatar.svg"></a>
 <a href="https://opencollective.com/bpkg/sponsor/8/website" target="_blank"><img src="https://opencollective.com/bpkg/sponsor/8/avatar.svg"></a>
 <a href="https://opencollective.com/bpkg/sponsor/9/website" target="_blank"><img src="https://opencollective.com/bpkg/sponsor/9/avatar.svg"></a>
-
+Editing bpkg/README.md at master · bpkg/bpkg
```

Comment:
what is this?

Reply:
@jwerle I don't know too, maybe it's a bug, I had feedback to https://github.community/t/edit-code-on-github-web-generate-strange-code/195323

I will edit and push again.